test/Microsoft.ML.AutoML.Tests/DatasetUtil.cs

```diff
@@ -203,7 +205,16 @@ private static void Download(string url, string destDir, string destFileName)
             if (File.Exists(relativeFilePath))
                 return;
 
-            new WebClient().DownloadFile(url, relativeFilePath);
+            using (var client = new HttpClient())
+            {
+                var response = await client.GetAsync(url);
```

Comment:
>response [](http://example.com/codeflow?start=20&length=8)

do we need to dispose this?

Reply:
Maybe?  I copied this from elsewhere in the repo which doesn’t dispose.  Perhaps disposing the wrapping client takes care of any open resources.  I will look into it and follow up with all the instances of the problem in the repo if necessary. 
https://github.com/dotnet/machinelearning/blob/e8073ad4eb59e0367a9c64f6f607f301282b2591/test/Microsoft.ML.Tokenizers.Tests/Utils.cs#L16
https://github.com/dotnet/machinelearning/search?q=httpclient+getasync&type=